src/Sarif.UnitTests/SarifCurrentToVersionOneTests.cs

```diff
@@ -923,6 +923,289 @@ public void SarifTransformerTests_ToVersionOne_NotificationExceptionWithStack()
             VerifyCurrentToVersionOneTransformation(V2LogText, V1LogExpectedText);
         }
 
+        [Fact]
+        public void SarifTransformerTests_ToVersionOne_ResultLocations()
+        {
+            const string V2LogText =
+@"{
+  ""$schema"": ""http://json.schemastore.org/sarif-2.0.0"",
+  ""version"": ""2.0.0"",
+  ""runs"": [
+    {
+      ""tool"": {
+        ""name"": ""CodeScanner""
+      },
+      ""logicalLocations"": {
+        ""html.body.header"": {
+          ""name"": ""header"",
+          ""decoratedName"": ""!!PageHeader#asdf"",
+          ""kind"": ""member"",
+          ""parentKey"": ""html.body""
+        },
+        ""html.body"": {
+          ""name"": ""body"",
+          ""kind"": ""namespace"",
+          ""parentKey"": ""html""
+        },
+        ""html"": {
+          ""name"": ""html"",
```

Comment:
`name` is not needed since it matches the key. It would be a good test to omit it and see if you still emit the correct SARIF v1. (Please take a look at the [SARIF v1 spec](https://rawgit.com/sarif-standard/sarif-spec/master/Static%20Analysis%20Results%20Interchange%20Format%20(SARIF).html) and see if you can omit t there too,).

Reply:
Fixed and confirmed